apps/condo/domains/ticket/components/TicketClassifierSelect.tsx

```diff
@@ -325,6 +325,8 @@ export const useTicketThreeLevelsClassifierHook = ({ initialValues: {
             { name: 'categoryClassifier', value: ruleRef.current.category },
             { name: 'problemClassifier', value: ruleRef.current.problem },
         ])
+        // the validation call is forced because antd does not make the field touched with setFields
```

Comment:
> ... antd does not make the field touched with setFields

Actually [it](https://ant.design/components/form/#FormInstance) [does](https://ant.design/components/form/#FieldData) :) But here we call validation programmatically according to our requirements.

Reply:
```suggestion
        // calling validation programmatically is necessary because antd does not mark the field as changed in `FieldData.touched` when using `setFields`
```